### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by purvasharmadev**
# Changes made:

> Added Simplified grid classes 
 - grid, grid-columns-two, grid-columns-three, grid-gap-1

## files to check

> grid.html, grid.css

### CODE REVIEW

**Review by adityapandadev — APPROVED** [review]
Add README File

File: Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.grid-columns-two{
+    grid-template-columns: 1fr 1fr;
+}
+
+.grid-gap-1{
+    gap: 1rem;
+}
+
+.grid-columns-three{
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (rutu14) [inline Components/Grid/grid.css]: Suggestion: can use the repeat statement, since the column size are same

**Review by rutu14 — APPROVED** [review]
It is good to merge.

File: Components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.grid .grid-ex{
+    height: 350px;
```
Comment (uday-nv-work) [inline Components/Grid/grid.css]: Suggestion: You can use rems or fr here instead of pixels

**Review by uday-nv-work — APPROVED** [review]
The code is great. The functionality works fine!
Good to merge.

**Pull request merged**